### ISSUE TITLE
[question] one-way copy possible? or master copy of inventory based on a server.

### ISSUE THREAD

**Issue by jplusc** [brunyman/MPDB#9]
Hello.   Thank you for making this plugin.

I wanted to know if 'MySQL Player Data Bridge' or 'MySQL Inventory Bridge' had the ability to copy the players inventory into the database when they left Server1 so they can have it on Server2 and Server3, but when they leave Server2 or Server3, don't update the inventory Server1 still has the same inventory.  (which when updated would be the new inventory copied to server2/server3)

please let me know if this configuration is possible.

thanks!

**Comment by brunyman** [comment]
it could be. but inventory on server 2 and 3 will be lost upon disconnection

**Comment by jplusc** [comment]
correct, any changes in server2/3 would be lost.  but when they connect again, they would have a copy of server1 inventory again.   or would they have to connect to server1 first?
